.github/workflows/pinned_build.yaml

```diff
@@ -26,3 +26,45 @@ jobs:
           platform-file: .cicd/platforms.json
           password: ${{secrets.GITHUB_TOKEN}}
           package-name: builders
+
+  Build:
+    name: Build
+    needs: [d]
```

Comment:
we're not using the cached build environments in this job, so you can completely remove the "Discover Platforms" job

Reply:
Yeah, I put that in simply to not have a completely empty initial workflow file in https://github.com/AntelopeIO/leap/pull/1290. I'll remove it here.

Reply:
Addressed: [7c10660](https://github.com/AntelopeIO/leap/pull/1291/commits/7c106605cc4ac138c3604c9fc659913a9e28cb4c)